larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -251,11 +251,8 @@ void StitchingCosmicRayMergingTool::CreatePfoMatches(const LArTPC &larTPC1, cons
         return;
     }
 
-    // To skip impact parameters
-    const bool isInGap3D_1(LArGeometryHelper::IsInGap(this->GetPandora(), pointingVertex1.GetPosition(),  TPC_VIEW_W, 0.f));
-    const bool isInGap3D_2(LArGeometryHelper::IsInGap(this->GetPandora(), pointingVertex2.GetPosition(),  TPC_VIEW_W, 0.f));
-
-    if (!isInGap3D_1 || !isInGap3D_2)
+    if (!LArGeometryHelper::IsInGap(this->GetPandora(), pointingVertex1.GetPosition(),  TPC_3D, 0.f) ||
+        !LArGeometryHelper::IsInGap(this->GetPandora(), pointingVertex2.GetPosition(),  TPC_3D, 0.f))
```

Comment:
(Could consider making gap tolerance configurable, but don't explicitly need to.) Could just drop the explicit 0.f, and the function should default this parameter to 0.f.